docs/design/libraries/ComInterfaceGenerator/DerivedComInterfaces.md

```diff
@@ -0,0 +1,191 @@
+# Derived Interfaces and COM
+
+In the ComInterfaceGenerator, we want to improve the experience when writing COM interfaces that derive from other COM interfaces. The built-in system has some quirks due to differences between how interfaces work in C# in comparison to COM interfaces in C++.
+
+## COM Interface Inheritance in C++
```

Comment:
Thoughts on adding the sections about how it works in C++ and .NET ComImport (and then GeneratedComInterface once it ships) to public docs? Probably somewhere under https://learn.microsoft.com/dotnet/standard/native-interop/cominterop

I think the descriptions you have here are clear and useful. I don't think we have this called out or explained anywhere in official docs.

Reply:
That's a good idea! I'll make sure to add it to the docs.